src/routes/lobby/LobbyView.vue

```diff
@@ -142,6 +147,9 @@ export default {
   },
   computed: {
     ...mapStores(useGameStore, useAuthStore),
+    showLobbyError() {
+      return this.$route.meta.error ? true : false; 
```

Comment:
Thoughts on ? vs !!   ?
```suggestion
      return !!this.$route.meta.error; 
```